### ISSUE TITLE
Full text search support

### ISSUE THREAD

**Issue by iloveitaly** [monicahq/monica#5079]
It would be great if it was possible to do a full text search on all data associated with contacts (e.g. notes).

The use case here is if you leave a note on a contact and later on you want to search for the contact that contains that keyword. For instance, let's say you met someone who is really into gardening, and then years later you want to start a garden: you'll want to search through your contacts for "gardening" but nothing will come up since the search bar only searches the name of the contact.

**Comment by OrinSafko** [comment]
@iloveitaly agreed. This would be extremely useful. Monica competitor Dex provides a universal search. It's extremely handy. I would love to see this in Monica.
